sdk/cosmos/ci.yml

```diff
@@ -6,7 +6,11 @@ resources:
     - repository: azure-sdk-build-tools
       type: git
       name: internal/azure-sdk-build-tools
-
+    - repository: azure-sdk-tools
+      type: github
```

Comment:
@chidozieononiwu looks like this one was overlooked in https://github.com/Azure/azure-sdk-for-java/pull/5314 could you please double check to see if there are any others you might have missed.

Reply:
Thanks for pointing it out. I will look through again.